t/app/controller/waste_merton.t

```diff
@@ -137,28 +144,54 @@ FixMyStreet::override_config {
         is $cgi->param('attribute[Action]'), '3';
         is $cgi->param('attribute[Reason]'), '2';
     };
+
+    my $echo_report;
```

Comment:
Ah, I see there's a flaky test - the new report being created here, when later on it calls send reports, that can be in any order (though normally is always the same) so might return this one rather than the new one last (to fill in Open311->test_req_used). So I think'll I just tweak this to delete the non-Echo report at the end of its test, then it's not there any more.